library/core/resource_loader/src/main/java/org/quiltmc/qsl/resource/loader/impl/ResourceLoaderImpl.java

```diff
@@ -523,4 +537,50 @@ public static void appendLanguageEntries(@NotNull Map<String, String> map) {
 			}
 		}
 	}
+
+	/* Static pack stuff */
+	private static List<ResourcePack> findUserStaticPacks() {
+		List<ResourcePack> returnList = new ArrayList<>();
```

Comment:
```suggestion
		List<ResourcePack> returnList = new ArrayList<>();

```

By the way, we allow the usage of `var` for variables that use `new`. Letting you the choice of using it or not as I feel like it'd be weird to nitpick ahah.